### PR TITLE
Bonsai skeleton implementation

### PR DESCRIPTION
**Pull request by IKnowLogic**
**Reason for pull request**
This pull request converts the Exiobase-conversion-software to the python skeleton application format. 

This enables us to use the conversion-software as a cli-tool.

**New files**:

`bin/csv2rdf_cli.py`: This is the entrance point for the csv2rdf cli tool. 
`bin/excel2csv_cli.py`: Entrance point to the excel2csv cli tool.

Since both tools offer many input arguments, I found it easier to create two separate cli tools, than one big.

`setup.py`: setupfile for easy installation of the tools. 

**Extensions**
This pull request also features a new argument for the `csv2rdf` cli tool, to split the output rdf graph into multiple smaller graphs of self specified size. This was introduced due to memory limitations when parsing large csv files. This option only works for .nt files.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+skip_commits:
```
Comment (kuzeko) [inline .appveyor.yml:1]: what is this file about? Is it ok that uses windows?

File: EXIOBASE_conversion_software/csv2rdf.py [hunk 2]
```diff
@@ -32,31 +32,56 @@
...
-    the dataframe is output of the `xslb2csv` function.
-    the code specifies which file is this taken from HSUP/HUSE/Other?
-    isInput=true -> the data is about input/use flows
-    isInput=false -> the data is about output/supply flows
+def serialize_graph(args, filename, graph, index=None):
+    outdir = args.outdir
+    if not outdir.endswith('/'):
+        outdir = outdir + '/'
 
-    # Data row:
```
Comment (kuzeko) [inline EXIOBASE_conversion_software/csv2rdf.py:51]: Has this documentation about the file structure been kept somewhere?

File: README.md [hunk 3]
```diff
@@ -6,26 +6,40 @@ This readme explains the reasoning behind the choice to use the hybrid (rather t
...
 
-### Convert `csv` to `ttl`
+
+## Usage
+### As a command line tool
+### Convert `xslb` to `csv`
 ```
-python scripts/csv2rdf.py -i data/MR_HSUP_2011_v3_3_17.csv -o data/  -c HSUP --flowtype output
-python scripts/csv2rdf.py -i data/MR_HUSE_2011_v3_3_17.csv -o data/  -c HUSE --flowtype input
+$ excel2csv-cli -i data/MR_HSUP_2011_v3_3_17.xlsb -o data/
```
Comment (kuzeko) [inline README.md:32]: could this be just `excel2csv` without `-cli` ? Same for the other command?

File: .appveyor.yml (hunk 1 above)
Comment (IKnowLogic) [reply .appveyor.yml:1]: @kuzeko I have no idea, this file was part of both the arborist script and the skeleton app, and it seemed nothing had to be changed.

File: README.md (hunk 3 above)
Comment (IKnowLogic) [reply README.md:32]: @kuzeko  We sure could, the `-cli` extension is however currently used for the Arborist script, so following that standard, cli tools should end with `-cli`.

File: .appveyor.yml (hunk 1 above)
Comment (kuzeko) [reply .appveyor.yml:1]: Interesting, let's assume it works then

File: README.md (hunk 3 above)
Comment (kuzeko) [reply README.md:32]: I see, ok then

File: EXIOBASE_conversion_software/csv2rdf.py (hunk 2 above)
Comment (IKnowLogic) [reply EXIOBASE_conversion_software/csv2rdf.py:51]: @kuzeko Yes, in the `makeRDF` method declaration.

**Pull request merged**